pkg/authorization/admission/restrictusers/subjectchecker.go

```diff
@@ -79,12 +79,8 @@ func NewRoleBindingRestrictionContext(ns string, kc kclientset.Interface, oc ocl
 }
 
 // labelSetForUser returns the label set for the given user subject.
-func (ctx *RoleBindingRestrictionContext) labelSetForUser(subject kapi.ObjectReference) (labels.Set, error) {
-	if subject.Kind == authorizationapi.SystemUserKind {
-		return labels.Set{}, nil
-	}
-
-	if subject.Kind != authorizationapi.UserKind {
+func (ctx *RoleBindingRestrictionContext) labelSetForUser(subject rbac.Subject) (labels.Set, error) {
```

Comment:
Are NotFoundErrors well tolerated by callers?

Reply:
Seem like all errors are tolerated equally as they are all printed and gathered in a composed error